.editorconfig

```diff
@@ -13,3 +13,16 @@ insert_final_newline = true
 charset = utf-8
 indent_style = space
 indent_size = 2
+
+[*.{kt,kts}]
+ktlint_standard_no-wildcard-imports = disabled
+ktlint_standard_no-consecutive-blank-lines = disabled
+ktlint_standard_final-newline = disabled
+ktlint_standard_filename = disabled
+ktlint_standard_property-naming = disabled
+ktlint_standard_class-naming = disabled
+ktlint_standard_no-consecutive-comments = disabled
+ktlint_standard_import-ordering = disabled
+ktlint_standard_function-naming = disabled
+ktlint_standard_backing-property-naming = disabled
+ktlint_standard_no-empty-file = disabled
```

Comment:
what is the need for disabling so many defaults?

Also, to have a less of the change in this PR due to re-formatting you can change the default style `intellij_idea` (see https://pinterest.github.io/ktlint/latest/rules/configuration-ktlint/#code-style), because Intellij IDEA built-in formatter was used for the majority of Kotlin files in this repo.